Dockerfile

```diff
@@ -28,68 +32,77 @@ RUN pip3 install bs4
 RUN yum -y install tk
 
 # Install Cloudera dependencies - required by IT Spark clusters
-RUN yum -y install \
-    alsa-lib \
-    at \
-    bc \
-    cronie \
-    cronie-anacron \
-    crontabs \
-    cvs \
-    db4-cxx \
-    db4-devel \
-    ed \
-    file \
-    gdbm-devel \
-    gettext \
-    jpackage-utils \
-    libXi \
-    libXtst \
-    man \
-    passwd \
-    pax \
-    perl-CGI \
-    perl-ExtUtils-MakeMaker \
-    perl-ExtUtils-ParseXS \
-    perl-Test-Harness \
-    perl-Test-Simple \
-    perl-devel \
-    redhat-lsb-core \
-    rsyslog \
-    time \
-    xz \
-    xz-lzma-compat
+RUN yum -y  install \
```

Comment:
Are these still needed according to Prasanth?

Reply:
Yes, he asked to have those for a bit longer, because not all clusters were updated.